config.py

```diff
@@ -33,32 +32,34 @@ class Config(object):
     DM_SUBMISSIONS_BUCKET = None
     DM_ASSETS_URL = None
 
+    DM_HTTP_PROTO = 'http'
+
     DEBUG = False
 
     RESET_PASSWORD_EMAIL_NAME = 'Digital Marketplace Admin'
-    RESET_PASSWORD_EMAIL_FROM = 'enquiries@digitalmarketplace.service.gov.uk'
+    RESET_PASSWORD_EMAIL_FROM = 'marketplace+password@digital.gov.au'
```

Comment:
The buyer frontend is just using a works-for-now value, but I changed this config to match it until we have something better.